minecraft/dial.go

```
@@ -194,31 +205,55 @@ func (d Dialer) DialContext(ctx context.Context, network, address string) (conn
 			if err != nil {
 				return nil, &net.OpError{Op: "dial", Net: "minecraft", Err: fmt.Errorf("create OIDC verifier: %w", err)}
 			}
+			if d.PlayFabClient == nil {
+				client, err := playfab.LoginWithXbox(ctx, e.PlayFabTitleID, d.XBLClient, playfab.ClientConfig{
+					HTTPClient:    d.HTTPClient,
+					CreateAccount: true,
+				})
+				if err != nil {
+					return nil, &net.OpError{Op: "dial", Net: "minecraft", Err: fmt.Errorf("login to playfab: %w", err)}
+				}
+				defer client.Close()
```

golangci-lint (2.12.1)

[error] Error return value of `client.Close` is not checked

(errcheck)